relay-event-normalization/src/normalize/processor.rs

```diff
@@ -878,19 +878,22 @@ fn normalize_performance_score(
                 if !profile
                     .score_components
                     .iter()
-                    .all(|c| measurements.contains_key(c.measurement.as_str()))
+                    .all(|c| measurements.contains_key(c.measurement.as_str()) || c.weight == 0.0)
```

Comment:
nit: Took me a while to understand that this means "all measurements with a profile weight greater zero are required". Might be worth adding a comment about it, or rewriting as `.any()` (flip the condition).

Reply:
Updated! I reversed the `all` to `any` and updated the comment